lib/pages/donate/donate.dart

```diff
@@ -3,11 +3,14 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
 class DonateRoute extends StatelessWidget {
+
+  final String _title = 'Kúpte nám kávu';
+
   @override
   Widget build(BuildContext context) {
-    String _title = 'Kúpte nám kávu';
     List<Iban> ibans = [
```

Comment:
But I think we need to come up with something better when we'll release the app, or remove it at all

Reply:
DB is the way :D